spec/controllers/api/v2/search_references_controller_spec.rb

```diff
@@ -58,11 +58,11 @@
       }
     end
 
-    it 'peforms lookup with letter A by default' do
-      get :index,
-          format: :json
+    it 'does not filter by letter' do
```

Comment:
I'd surround this in a context as it doesn't make sense without one

Reply:
Ignore this, just looked at the file the previous example description was adding context it shouldn't have

Reply:
it's already inside  the context 'with no letter param provided' do
what we do not need is the pattern since this is just checking the counts.